db/views/latest_blood_pressures_per_patient_per_months_v06.sql

```diff
@@ -0,0 +1,20 @@
+SELECT DISTINCT ON (patient_id, year, month)
+    blood_pressures.id AS bp_id,
+    blood_pressures.patient_id AS patient_id,
+    patients.registration_facility_id AS registration_facility_id,
+    patients.assigned_facility_id AS assigned_facility_id,
+    patients.status as patient_status,
+    blood_pressures.facility_id AS bp_facility_id,
+    blood_pressures.recorded_at AT TIME ZONE 'UTC' AT TIME ZONE 'UTC' AS bp_recorded_at,
```

Comment:
Here is how we maintain the timestamps as UTC when grabbing them from the application tables.  

We need to be very explicit here and tell PostgreSQL we are grabbing `bp_recorded_at` UTC and storing It as UTC. if we don't do this, the PSQL TimeZone setting may break things by silently converting.  I originally tried a straight import without any time zone conversion at all, but that was causing issues and also was scary with how much magical timezone conversion could be happening.

Reply:
This shouldn't behave differently for app tables and reporting tables no?

>  PSQL TimeZone 

Are you referring to the psql client timezone?

Reply:
> This shouldn't behave differently for app tables and reporting tables no?

In the prod app today, it doesn't behave differently between app and reporting tables, because:

* application rows are written with the PSQL session time zone set to UTC and the Rails app time zone set to UTC (ie the default)
* reporting rows are written with the PSQL session time zone set to UTC (I believe we don't change this) and the app time zone set to REPORTING_TIME_ZONE (ie IST in India)

Given the differences in these two contexts, I wanted to be 100% explicit and clear when creating the materialized view.